sentiment-api/service.bal

```diff
@@ -0,0 +1,56 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/openapi;
+
+@openapi:ServiceInfo {
```

Comment:
Shall we remove the openapi annotations to keep it simple?

Reply:
Fixed via [fdc5c9c](https://github.com/TharmiganK/ieee-hackathon-workshop/pull/1/commits/fdc5c9cb6fc7fb34e59140109996174f90abb924)